packages/sw-cache-expiration/src/lib/error-factory.js

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
We could, but let's just do that in a separate PR that covers the entire set of modules if we care.